[rospy.client][INFO] 2021-09-07 11:04:28,629: init_node, name[/server_endpoint], pid[10598]
[xmlrpc][INFO] 2021-09-07 11:04:28,630: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-09-07 11:04:28,630: Started XML-RPC server [http://wng:46307/]
[rospy.init][INFO] 2021-09-07 11:04:28,630: ROS Slave URI: [http://wng:46307/]
[rospy.impl.masterslave][INFO] 2021-09-07 11:04:28,630: _ready: http://wng:46307/
[xmlrpc][INFO] 2021-09-07 11:04:28,630: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2021-09-07 11:04:28,630: Registering with master node http://localhost:11311
[rospy.init][INFO] 2021-09-07 11:04:28,730: registered with master
[rospy.rosout][INFO] 2021-09-07 11:04:28,731: initializing /rosout core topic
[rospy.rosout][INFO] 2021-09-07 11:04:28,732: connected to core topic /rosout
[rospy.simtime][INFO] 2021-09-07 11:04:28,732: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-09-07 11:04:28,737: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-09-07 11:04:28,974: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-09-07 11:04:58,425: Connection from 192.168.1.146
[rosout][INFO] 2021-09-07 11:04:58,430: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-09-07 11:04:58,478: topic[/execution_check] adding connection to [http://wng:46307/], count 0
[rospy.internal][INFO] 2021-09-07 11:04:58,479: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-09-07 11:21:57,050: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-09-07 11:21:57,053: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-09-07 11:21:57,053: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-09-07 11:21:57,054: topic[/execution_check] removing connection to http://wng:46307/
[rospy.impl.masterslave][INFO] 2021-09-07 11:21:57,054: signal-2
[rospy.internal][WARNING] 2021-09-07 11:21:57,054: Unknown error initiating TCP/IP socket to wng:40415 (http://wng:46307/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.core][INFO] 2021-09-07 11:21:57,161: signal_shutdown [atexit]
